core/src/main/java/org/apache/iceberg/CatalogProperties.java

```diff
@@ -160,4 +160,10 @@ private CatalogProperties() {}
 
   public static final String ENCRYPTION_KMS_TYPE = "encryption.kms-type";
   public static final String ENCRYPTION_KMS_IMPL = "encryption.kms-impl";
+  public static final String WRITER_KEK_TIMEOUT_SEC = "encryption.kek-timeout-sec";
```

Comment:
Avoid using `kek` because it is not clear. Is there a good term for the key you're referring to here?

Reply:
Since we have a reasonably well defined value of this parameter in the NIST spec, maybe we don't need to make it configurable in an initial version of table encryption. This parameter is also not easy to explain. So I'll remove this from the `CatalogProperties` for now. If, for some reason, a requirement comes later to make this configurable, we can this back, with a proper name and documentation.

Reply:
Sounds great!